.github/ISSUE_TEMPLATE/security-vulnerability.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Security vulnerability
+about: 'View our security policy here: https://curl.haxx.se/docs/security.html'
+title: 'View our security policy here: https://curl.haxx.se/docs/security.html'
+labels: ''
+assignees: ''
+
+---
+
+View our security policy here: https://curl.haxx.se/docs/security.html
+
+DO NOT REPORT THE ISSUE PUBLICLY ON GITHUB
```

Comment:
Wouldn't it be better if this instead linked directly to the hackerone page where user would submit the security issue? https://hackerone.com/curl